apps/api/src/account/account.service.ts

```diff
@@ -1,4 +1,12 @@
 import { Injectable } from '@nestjs/common'
+import { UpdatePersonDto } from '../person/dto/update-person.dto';
+import { PrismaService } from '../prisma/prisma.service';
 
 @Injectable()
-export class AccountService {}
+export class AccountService {
+    constructor(private prismaService: PrismaService) {}
+
+    async updateUserProfile(id: string, data: UpdatePersonDto) {
```

Comment:
Let's add type-hint of the resulting object

```suggestion
    async updateUserProfile(id: string, data: UpdatePersonDto): Promise<Person> {
```